### PR TITLE
Update docs on local E2E setup 

### PR DESCRIPTION
**Pull request by MaggieSalak**
# PR for issue #751 

## What is being addressed

While trying to set up the E2E tests to run locally on my LoRa gateway and leaf device, I found a few gaps that in my view are missing from the current E2E readme, focusing on the requirements to get the tests to run locally (LBS and LNS).

## How is this addressed

E2E tests readme was updated with the steps I found out about while working on running the tests locally.

### CODE REVIEW

File: Tests/E2E/README.md [hunk 1]
```diff
@@ -18,17 +18,54 @@ This guide helps you to execute and author E2E tests on your local environment.
...
+    - Update LoRaWanNetworkSrvModule settings:
+        ```
+        NET_SRV_LOG_LEVEL=1
+        NET_SRV_LOGTO_HUB=true
+        NET_SRV_LOGTO_UDP=true
+        NET_SRV_LOG_TO_UDP_ADDRESS=<your-local-ip-address>
+        ```
+    - Update `LBS_TC_URI`:
+    
+        `LBS_TC_URI=ws://172.17.0.1:5000`
```
Comment (MaggieSalak) [inline Tests/E2E/README.md:47]: @Mandur I'm not 100% if there was another place where this Uri needed to be configured, do you know?

File: LoRaEngine/deployment.lbs.template.json [hunk 2]
```diff
@@ -107,6 +107,9 @@
                 },
                 "APPINSIGHTS_INSTRUMENTATIONKEY": {
                   "value": "$APPINSIGHTS_INSTRUMENTATIONKEY"
+                },
+                "REGION": {
```
Comment (MaggieSalak) [inline LoRaEngine/deployment.lbs.template.json:111]: Without Region in the deployment file I was seeing errors in the LBS so I thought we should commit it to the template as well, any reason we shouldn't have it here @Mandur ?

**Comment by roel4ez** [comment]
added #803

File: LoRaEngine/deployment.lbs.template.json (hunk 2 above)
Comment (Mandur) [reply LoRaEngine/deployment.lbs.template.json:111]: To be honest I think this was needed before @bastbu added the dynamic configuration. I think it is not needed anymore (at least CI is not populating it), could you please retry on latest dev?

File: Tests/E2E/README.md [hunk 3]
```diff
@@ -18,17 +18,54 @@ This guide helps you to execute and author E2E tests on your local environment.
...
+
+2. Build and push IoT Edge solution to your container registry using `LoRaEngine/deployment.lbs.template.json` file.
+
+    In VS Code: Ctrl+Shift+P -> Azure IoT Edge: Bild and Push IoT Edge Solution -> select template file. 
+
+3. Make sure the generated deployment template (`LoRaEngine/config/deployment.lbs.json`) contains the correct region configuration in the `LoRaWanNetworkSrvModule` section, e.g.:
+    ```
+    "REGION": {
+        "value": "EU868"
```
Comment (danigian) [inline Tests/E2E/README.md:56]: Ditto as Mik comment from before, this should not be needed anymore on latest dev

File: LoRaEngine/deployment.lbs.template.json (hunk 2 above)
Comment (MaggieSalak) [reply LoRaEngine/deployment.lbs.template.json]: Verified and indeed the REGION config is not needed.

File: Tests/E2E/README.md [hunk 4]
```diff
@@ -18,17 +18,47 @@ This guide helps you to execute and author E2E tests on your local environment.
...
   * LOG_TO_UDP_ADDRESS: development machine IP address (ensure IoT Edge machine can ping it)
 * E2E test configuration (in file `appsettings.local.json`) has UDP logging enabled `"UdpLog": "true"`
 
-## Installation
+## Setup
 
-1. Connect and setup Seeduino Arduino with the serial pass sketch
+### LoRa Basics Station and LNS setup
+
+Configure LoRa Basics Station and Network Server to run on your local LoRa gateway:
```
Comment (p-schuler) [inline Tests/E2E/README.md:33]: By local LoRa gateway you mean what exactly? I don't think it's clear to the reader. I assume you mean the concentrator and network server running on your local machine?

Comment (MaggieSalak) [reply Tests/E2E/README.md:33]: It means both components running on the concentrator. I will update this.

**Comment by MaggieSalak** [comment]
Closing this PR as another one has been opened which migrates this document to MkDocs: https://github.com/Azure/iotedge-lorawan-starterkit/pull/814

**Pull request closed without merging**